diff --git a/node_modules/react-native-iap/ios/IapSerializationUtils.swift b/node_modules/react-native-iap/ios/IapSerializationUtils.swift
--- a/node_modules/react-native-iap/ios/IapSerializationUtils.swift
+++ b/node_modules/react-native-iap/ios/IapSerializationUtils.swift
@@ -204,12 +204,6 @@
         "webOrderLineItemID": t.webOrderLineItemID
     ]
 
-    #if compiler(>=5.10)
-    if #available(iOS 15.2, tvOS 15.2, *) {
-        result["appTransactionID"] = t.appTransactionID
-    }
-    #endif
-
     return result
 }
 @available(iOS 15.0, tvOS 15.0, *)
